toolkits/global/packages/global-expander/js/expander.js

```diff
@@ -41,7 +41,7 @@ const Expander = class {
 		if (this._isOpen) {
 			this._close();
```

Comment:
should this not be `close()` now (and a bit further down)?

Reply:
Whoops. I thought I'd don't a find and replace but clearly hadn't. Thanks